test/function/samples/plugin-hook-filters/_config.js

```diff
@@ -0,0 +1,157 @@
+const assert = require('assert');
+const util = require('util');
+const path = require('path');
+
+const idFilterHooks = ['resolveId', 'load'];
+const hooks = [...idFilterHooks, 'transform'];
+
+const calledHooks = {};
+for (const hook of hooks) {
+	calledHooks[hook] = {};
+}
+
+const s = {
```

Comment:
Maybe call this something descriptive like "expectedCalledHooks"? But I must admit, this is a rather thorough and cool test.